src/App.tsx

```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
+// tslint:disable: ordered-imports
 import MainPage from "./components/mainPage/mainPage";
 import Navigation from "./components/navigationBar/navigationBar";
 import ProofEditor from "./components/proofEditor/proofEditor";
+import Einstellungen from "./components/einstellungen/einstellungen";
+import Kontakt from "./components/kontakt/kontakt";
```

Comment:
What is the difference between `Kontakt` and `Impressum`?
Online I found this http://www.impressum-recht.de/impressum-kontakt-angaben.html.

At the bottom of the page it says `als Bezeichnung bieten sich "Impressum" oder "Kontakt" an;`,
so it seems like these two words are synonyms.

Reply:
I think you should see them as dummy until we made a impressum or contact page. Actually you can see how the menu works. We can add the impressum/contact page later but in this version we should focus of the functionality of the basic elements.

Reply:
I just implemented what was given before so no design choice on my side. Though I think sometimes there can be an impressum and a contact page and the impressum would hold the contact information while the contact page has a formular which will send an email to the site owner or support.

But we will see what we need later and it shouldn't be hard to change it. So as @RonjaKaehne says they are pretty much dummys for now.